src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,41 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+public class Contact {
+    /**
+     * Base Class to build a Person's phone/email/address from.
```

Comment:
No need to mention these details here. 
Also, this comment is better outside the class definition